### PR TITLE
Prepare version 6.3.1

### PR DESCRIPTION
**Pull request by j-rivero**
# 🎈 Release

Preparation for 6.3.1 release.

Comparison to 6.3.0: https://github.com/ignitionrobotics/sdformat/compare/sdformat6_6.3.0...sdf6

## Checklist
- [x] Asked team if this is a good time for a release
- [x] There are no changes to be ported from the previous major version
- [x] No PRs targeted at this major version are close to getting in
- [x] Bumped minor for new features, patch for bug fixes
- [x] Updated changelog

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Review by azeey — APPROVED** [review]

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 ## SDFormat 6.0
 
-### SDFormat 6.X.X (20XX-XX-XX)
+### SDFormat 6.3.1 (2021-07-06)
+
+1. Fix flattening logic for nested model names
+    * [Pull request 597](https://github.com/osrf/sdformat/pull/597)
+
+1. Fix flattening logic for nested model names
```
Comment (scpeters) [inline Changelog.md:8]: fix title: "Translate poses of nested models inside other nested models"

**Review by chapulina — APPROVED** [review]

**Review by scpeters — APPROVED** [review]

File: Changelog.md (hunk 1 above)
Comment (j-rivero) [reply Changelog.md]: thanks!

**Comment by chapulina** [comment]
Ouch, this branch doesn't have GitHub actions setup. Our options:

1. Set it up copying from newer branches
1. Ignore it and merge this with admin override
1. Ignore it the proper way through special settings for this branch

I'm in favor of 1

**Comment by j-rivero** [comment]
> Ouch, this branch doesn't have GitHub actions setup. Our options:
> 
>     1. Set it up copying from newer branches

In this same PR? If it is just copy the setup from sdf9 or newer I can do it yes.

**Comment by chapulina** [comment]
> In this same PR?

I'm ok with that

**Pull request merged**